bodhi/client/__init__.py

```diff
@@ -79,8 +89,20 @@ def new(user, password, **kwargs):
             print_resp(resp, client)
         except bindings.BodhiClientException as e:
             click.echo(str(e))
-        except Exception as e:
-            traceback.print_exc()
```

Comment:
Let's not make this change in this commit, since it's unrelated to adding an edit command.